web/src/main/js/util/configurationRequestBuilder.js

```diff
@@ -37,52 +37,52 @@ export function createReadRequest(apiUrl, csrfToken, configurationId = null) {
     if (configurationId) {
         url = url.concat(`/${configurationId}`);
     }
+    const headers = new Headers();
+    headers.append('X-CSRF-TOKEN', csrfToken);
     return fetch(url, {
         credentials: 'same-origin',
-        headers: {
-            'X-CSRF-TOKEN': csrfToken
-        }
+        headers
     });
 }
 
 export function createNewConfigurationRequest(apiUrl, csrfToken, fieldModel) {
+    const headers = new Headers();
+    headers.append('Accept', 'application/json');
+    headers.append('Content-Type', 'application/json');
+    headers.append('X-CSRF-TOKEN', csrfToken);
```

Comment:
You could also create constant header objects in the header class that will return this grouping of headers so you don't have to repeat it every time.

Reply:
The function would need to take in a csrfToken though.